tensorflow/contrib/tensorrt/BUILD

```diff
@@ -171,6 +184,25 @@ tf_py_wrap_cc(
     ],
 )
 
+tf_cuda_library(
+    name = "trt_resources",
+    srcs = [
+        "resources/TRTInt8Calibrator.cc",
+        "resources/TRTResourceManager.cc",
+    ],
+    hdrs = [
+        "resources/TRTInt8Calibrator.h",
+        "resources/TRTResourceManager.h",
+        "resources/TRTResources.h",
+    ],
+    deps = [
+        "@local_config_tensorrt//:nv_infer",
```

Comment:
Please use if_tensorrt() for anything that is related to nv_infer or cuda.

Reply:
done